src/main/scala/nl/biopet/tools/bamstats/schema/Data.scala

```diff
@@ -55,4 +55,23 @@ case class Data(flagStats: FlagStats,
       }
     }
   }
+
+  def validate(): Unit = {
```

Comment:
Not sure this this part should be part of this pull request, this does not feel like finished yet